expressmailing.php

```diff
@@ -58,7 +61,7 @@ public function install($alter_db = true)
 		return parent::install()
 			&& $this->installDB($alter_db)
 			/* Add main Tab (visible into Customer) */
-			&& $this->installAdminTab('AdminMarketingX', true, $this->displayName, 'AdminParentModules')
+			&& $this->installAdminTab('AdminMarketingX', true, $this->displayName, 'AdminPriceRule')
```

Comment:
If you want to move it on the shops where the module is already installed, you have to implement an upgrade file.

http://doc.prestashop.com/display/PS15/Auto-updating+modules